test/testConstants/PaymentAllocationsTestConstants.scala

```diff
@@ -35,6 +35,7 @@ object PaymentAllocationsTestConstants {
     outstandingAmount = Some(-200.00),
     documentDate = LocalDate.of(2018, 3, 29),
     paymentLot = Some("paymentLot"),
+    effectiveDateOfPayment = Some(LocalDate.parse("2021-01-31")),
```

Comment:
pls use something similar to the lines above: LocalDate.of(2018, 3, 29) / avoid data parsing